environment.yml

```diff
@@ -22,7 +22,7 @@ dependencies:
 - scipy
 - jupyterlab
 - matplotlib
-- seaborn
+- seaborn==0.10.1
```

Comment:
Why is this needed?

Reply:
The jointplot function inputs were changed in 0.11 and produce an error, I couldn't find the argument in question(stat_func=None) in the docs however, so I just set it to to 0.10.1

Reply:
Okay thanks, I was just curious :)